tests/unit/conftest.py

```diff
@@ -834,6 +835,21 @@ def record_with_files_creation(db, record_with_bucket):
     return pid, record, record_url
 
 
+@pytest.fixture
+def record_with_image_creation(db, record_with_bucket):
+    """Creation of a full record with files in database."""
+    pid, record = record_with_bucket
+    filename = 'Test.png'
+    with open(os.environ['INVENIO_STATIC_FOLDER'] + '/img/eu.png', 'r') as f:
```

Comment:
It's better to use the `zenodo` Python package files via `pkg_resource.resource_filename` to get the file